src/features/tokens/balances.ts

```diff
@@ -90,8 +92,10 @@ export function useDestinationBalance(
       const route = getTokenRoute(originCaip2Id, destinationCaip2Id, tokenCaip19Id, tokenRoutes);
       const protocol = getProtocolType(destinationCaip2Id);
       if (!route || !recipientAddress || !isValidAddress(recipientAddress, protocol)) return null;
-      const adapter = AdapterFactory.HypTokenAdapterFromRouteDest(route);
-      const balance = await adapter.getBalance(recipientAddress);
+      const tokenAdapter = isIbcOnlyRoute(route)
```

Comment:
Not sure if it's relevant, but if we can cut complexity by not supporting the underlying IBC transfer, we probably should. I don't believe folks will be using the UI for that purpose

Reply:
Not really, that's why I added both